modules/exploits/unix/ssh/tectia_passwd_changereq.rb

```diff
@@ -189,8 +190,9 @@ def init_ssh(user)
     opts = {
       :user            => user,
       :port            => rport,
-      :verify_host_key => :never
-    }
+    }.merge(
```

Comment:
```suggestion
    opts = ssh_client_defaults.merge({
      :user            => user,
      :port            => rport
    })
```

We'd want to ensure we're overriding the defaults 👀